lib/src/test/java/uk/gov/di/ipv/core/library/domain/ContraIndicationsTest.java

```diff
@@ -4,9 +4,7 @@
 import org.junit.jupiter.api.Test;
 
 import java.time.Instant;
-import java.util.HashMap;
-import java.util.Map;
-import java.util.Optional;
+import java.util.*;
```

Comment:
IDE has wildcarded the import - Settings -> Editor -> Code Style -> Java -> Class count to use import with * and Names count to use static import with * - Set both these to a high number.
We shouldn't be using wildcard imports